lib/main.ts

```diff
@@ -156,9 +156,15 @@ export class Transpiler {
 
   // Visible for testing.
   getOutputPath(filePath: string, destinationRoot: string): string {
-    var relative = this.getRelativeFileName(filePath);
-    var dartFile = relative.replace(/.(js|es6|ts)$/, '.dart');
-    return path.join(destinationRoot, dartFile);
+    var dartFile = filePath.replace(/.(js|es6|ts)$/, '.dart');
```

Comment:
This part of the PR was in fact to solve a use case I found out during investigation:
`chai.expect(getNormalizedPath(transpiler, '/a/b/c.js', '/e/f/g/h/i/j/k/l')).to.equal('/a/b/c.dart');`

This is not a Windows thing, it was failing on all platforms.
To make things easier, I'll split the PR and try find a proper way to solve it.